extensions/amp-viewer-assistance/0.1/amp-viewer-assistance.js

```diff
@@ -70,7 +77,8 @@ export class AmpViewerAssistance {
    */
   actionHandler_(invocation) {
     const {method, args} = invocation;
-    if (method == 'updateActionState' && !!args) {
+    if (method == 'updateActionState' && args
+        && this.isValidActionStatusArgs_(args)) {
```

Comment:
Nit: Move `args && this.isValidActionStatusArgs_(args)` into a nested `if` to avoid the `else` branch triggering due to false condition.

```js
if (method == 'updateActionState') {
  if (args && this.isValidActionStatusArgs_(args) {
    ...
  }
} else if (method == 'signIn') {
```